VirtualEconomyFramework/VEDriversLite/NFT/Coruzant/CoruzantNFTHelpers.cs

```diff
@@ -44,62 +44,7 @@ public static async Task<CoruzantProfileNFT> FindCoruzantProfileNFT(ICollection<
             return new CoruzantProfileNFT("");
         }
 
-        /// <summary>
-        /// This function will new Coruzant Post NFT.
-        /// </summary>
-        /// <param name="address">sender address</param>
-        /// <param name="ekey">Encryption Key object of the address</param>
-        /// <param name="NFT">Input NFT object with data to save to metadata</param>
-        /// <param name="nutxos">List of spendable neblio utxos if you have it loaded.</param>
-        /// <param name="tutxos">List of spendable token utxos if you have it loaded.</param>
-        /// <returns>New Tx Id Hash</returns>
-        public static async Task<string> MintCoruzantPostNFT(string address, int coppies, EncryptionKey ekey, INFT NFT, string profileUtxo, ICollection<Utxos> nutxos, ICollection<Utxos> tutxos)

```

Comment:
Why have you deleted these functions. Could you please return them back?

Reply:
The methods were not having any reference so I have removed them as part of refactoring. Will add them back.

Reply:
you can comment them instead remove. They are prepared for some updates, but maybe it will be replaced with system of INFTModules implemented in #33 now.